agent/php_newrelic.h

```diff
@@ -460,6 +461,18 @@ nrinitime_t
     agent_span_queue_timeout; /* newrelic.infinite_tracing.span_events.agent_queue.timeout
                                */
 
+/*
+ * configuration options for logging in context
```

Comment:
I think it's better to leave 'logging in context' out from the code. How about this:
```suggestion
 * configuration options for handling application logging
```

Reply:
fixed in 8f2c5758ac268ad0ac61e8b726eabd3a10de9652